README.md

```diff
@@ -199,38 +199,29 @@ For an overview of the full workflow, see the [workflow diagram](./docs/workflow
    (venv) $ ilab
    Usage: ilab [OPTIONS] COMMAND [ARGS]...
 
-   CLI for interacting with InstructLab.
+     CLI for interacting with InstructLab.
 
-   If this is your first time running InstructLab, it's best to start with `ilab config init` to create the environment.
+     If this is your first time running ilab, it's best to start with `ilab config init` to create the environment.
```

Comment:
I would leave this as InstructLab and not ilab.

Reply:
Will do, thank you.